Userland/ls.cpp

```diff
@@ -156,7 +160,10 @@ bool print_filesystem_object(const char* path, const char* name) {
     else
         printf("%c", st.st_mode & S_IXOTH ? 'x' : '-');
 
-    printf(" %4u %4u", st.st_uid, st.st_gid);
+//    printf(" %4u %4u", st.st_uid, st.st_gid);
+    pw = getpwuid(st.st_uid);
```

Comment:
Both `getpwuid()` and `getgrgid()` can fail, and will then return null.
This code needs to handle that scenario, since a file can be owned by a UID/GID that doesn't exist in `/etc/passwd` and/or `/etc/group`.